packages/core/src/components/axes/zero-line.ts

```diff
@@ -7,12 +7,29 @@ export class ZeroLine extends Component {
 	type = "zero-line";
 
 	render(animate: boolean) {
+		const [minDomainValue, maxDomainValue] = this.services.cartesianScales.getRangeScale().domain();
+		const containsZero = (minDomainValue * maxDomainValue) < 0;
```

Comment:
hmmm... would this be true if domain is [0, 200]? do we care about that case?

Reply:
If domain is `[0, 200]`, `containsZero` will be false and I think that in that case is ok to not show the zero axis line because it coincides with the x axis.
But I can make it clearer replacing `(minDomainValue * maxDomainValue) < 0` with `minDomainValue > 0 && maxDomainValue < 0 || minDomainValue < 0 && maxDomainValue > 0`

Reply:
ok, technically the domain contains a zero at that point, so should we rename `containsZero` to `drawZeroLine`?

Reply:
I absolutely agree with you